pydis_site/apps/content/resources/guides/pydis-guides/contributing/installing-project-dependencies.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Installing Project Dependencies
+description: A guide to installing the dependencies of our projects.
+icon: fab fa-python
+---
+
+> **Note:** The process varies depending on your choice of code editor / IDE, so refer to one of the following guides:
+
+- [Installing dependencies with PyCharm](#installing-dependencies-with-pycharm)
+- [Installing dependencies with the command line](#installing-dependencies-with-the-command-line)
```

Comment:
Would this be better as a table of contents or would that make the note confusing?